Code2CollegeBankingProject/BankingUI/views.py

```diff
@@ -214,14 +213,15 @@ def deleteaccount(request):
                 "response": "Your account has been deleted.. along with all your money"
             }
             request.user.delete()
-            return render(request, "index.html", userinfo)
+            return redirect(index, userinfo)
```

Comment:
This is a good use of redirect because the are wanting the user to go from `/BankingUI/deleteaccount` to `BankingUI/index`

Reply:
done